packages/react-storage/src/components/StorageBrowser/Views/CreateFolderActionView/Controls.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Controls } from '../Controls';
+import { StorageBrowserElements } from '../../context/elements';
+
+export interface CreateFolderActionViewControls<
+  T extends StorageBrowserElements = StorageBrowserElements,
+> extends Pick<Controls<T>, 'Target'> {
+  (): React.JSX.Element;
+}
+
+const { Target } = Controls;
+
+// @ts-expect-error TODO: add Controls assignment
+export const CreateFolderActionViewControls: CreateFolderActionViewControls =
+  () => {
+    return (
+      <Target.Field.Container>
+        <Target.Field.Label>Enter folder name:</Target.Field.Label>
+        <Target.Field.Input type="text" />
+        <Target.Field.Submit
+          onClick={() => {
+            // eslint-disable-next-line no-console
+            console.log('create folder');
+          }}
+        >
+          Create folder
+        </Target.Field.Submit>
+      </Target.Field.Container>
```

Comment:
Am I using this right? I couldn't add some extra HTML elements to the `label` and `input` that I wanted to for accessibility. I was trying to do this:

```tsx
<form
  onSubmit={(event) => {
    // @TODO: we will probably want to use context and handleUpdateState()
    // to send the folder name to so that it can be sent off to be created?

    event.preventDefault();

    // eslint-disable-next-line no-console
    console.log('create folder', folderName);
  }}
>
  <label htmlFor="folder-name-input">Enter folder name:</label>
  <input
    type="text"
    id="folder-name-input"
    value={folderName}
    onChange={(event) => setFolderName(event.target.value)}
    required
  />
  <button type="submit">Create folder</button>
</form>
```

For instance, for `htmlFor`, typescript was complaining:
```
Type '{ children: string; htmlFor: string; }' is not assignable to type 'IntrinsicAttributes & Omit<BaseElementProps<never, string, DetailedHTMLProps<LabelHTMLAttributes<HTMLLabelElement>, HTMLLabelElement>>, "ref"> & RefAttributes<...>'.
  Property 'htmlFor' does not exist on type 'IntrinsicAttributes & Omit<BaseElementProps<never, string, DetailedHTMLProps<LabelHTMLAttributes<HTMLLabelElement>, HTMLLabelElement>>, "ref"> & RefAttributes<...>'.
  ```